### PR TITLE
Cypress for Integration

### PR DESCRIPTION
**Pull request by taymoork2**
Replace jest with cypress for integration tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
     "start": "nodemon -r dotenv/config scripts/start.js",
     "serve": "parcel scripts/index.html",
-    "test": "jest",
-    "test:integration": "jest \"^(?=.*integration.test.js).*\"",
-    "test:unit": "jest \"^(?=.*.test.js)(?!.*integration).*\"",
+    "jest": "jest",
+    "cypress": "cypress open",
+    "test:integration": "cypress run --browser chrome",
+    "test:integration:ci": "npm run test:integration -- --reporter junit --reporter-options \"mochaFile=test_results/cypress/results-[hash].xml,toConsole=true\"",
+    "test:unit": "jest",
```
Comment (taymoork2) [inline package.json:18]:
```suggestion
    "test:unit": "jest --silent",
```
this should be silent by default to be honest to hide all the console.errors from tests that are testing for errors

File: .gitignore [hunk 2]
```diff
@@ -39,9 +40,8 @@ reports/
 ._*
 
 #Local stuff
-*.json
 .env*
-!.env.default
-!package.json
```
[the remark this answers is not in the document]
Comment (taymoork2) [reply .gitignore:45]: because we're no longer ignoring all json files

**Review by jonniespratley — APPROVED** [review]
👍

**Pull request merged**